### DISCUSSION TITLE
Fetching Data for Prior Weeks (Basketball)

### DISCUSSION THREAD

**Discussion by itsdaveedwards** [cwendt94/espn-api#443] [Q&A]
Hey @cwendt94 thank you for all of your hard work on this API, I used it to build a iOS app for our league this year since we use custom scoring which ESPN doesn't support. 

I am trying to double check our ESPN data for the year before playoffs start (to account for retroactive stat changes that occur after each week of games ends, which my app was not set to check), but having a lot of issues trying to fetch weeks other than the current one.

Anytime I try to run some variant of 
box = league.box_scores(matchup_period = 2, scoring_period = 7, matchup_total = True)
(adding and subtracting the arguments, changing values, etc), I am just getting a list of matchups back but no box scores that my script can use. 

Am I doing something wrong? Running box = league.box_scores() always works fine, but it seems like maybe the data structure changes for prior weeks?

**Comment by cwendt94** [comment]
Hey @itsdaveedwards, what is the scoring type for your league? Also are you trying to get the matchup data or individual days in that matchup period? 

In my H2H points league I was able to get all of the second matchup data for from this call `league.box_scores(matchup_period = 2)`, If you want to get individual days with scoring period make sure to switch `matchup_total` to false,  `league.box_scores(matchup_period = 2, scoring_period = 7, matchup_total = False)`
